lib/espMqttClient/src/espMqttClient.cpp

```diff
@@ -78,37 +78,27 @@ espMqttClientSecure::espMqttClientSecure(uint8_t priority, uint8_t core)
 }
 
 espMqttClientSecure & espMqttClientSecure::setInsecure() {
-#if defined(EMC_CLIENT_SECURE)
```

Comment:
dows not compile with tasmota SDK if this is removed

Reply:
already fixed that this morning